norlab_icp_mapper/MapperModules/ComputeDynamicsMapperModule.cpp

```diff
@@ -0,0 +1,166 @@
+#include <nabo/nabo.h>
+#include "ComputeDynamicsMapperModule.h"
```

Comment:
Can this be renamed to ComputeDynamicPointsMapperModule or DynamicPointMapperModule?